tests/integration/serve_tests.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+use std::io::Read;
+
+use deno_fetch::reqwest;
+use pretty_assertions::assert_eq;
+use regex::Regex;
+use test_util as util;
+
+#[tokio::test]
+async fn deno_serve_port_0() {
```

Comment:
Could this be made into a spec test?

Reply:
No, we can't read the output of a spec test and then forward it to another test step